WalletWasabi.Fluent/Views/Wallets/Send/SendView.axaml

```diff
@@ -108,6 +108,26 @@
                                   IsConversionReversed="{Binding ConversionReversed, Mode=TwoWay}" />
         </DockPanel>
       </DockPanel>
+
+      <!-- Funding mode -->
+      <StackPanel Orientation="Horizontal" HorizontalAlignment="Right" Margin="0 -38 0 0" DockPanel.Dock="Top" Opacity="0.6" Spacing="5">
+        <PathIcon Data="{StaticResource info_regular}" Width="14" VerticalAlignment="Center" />
+        <Button Classes="activeHyperLink FontSizeH9" Command="{Binding FundingModeCommand}">
+          <Panel>
+            <Panel.Styles>
+              <Style Selector="Panel:pointerover > TextBlock">
+                <Setter Property="TextDecorations" Value="Underline" />
+              </Style>
+              <Style Selector="TextBlock">
+                <Setter Property="FontSize" Value="{StaticResource FontSizeH8}" />
+                <Setter Property="VerticalAlignment" Value="Center" />
+              </Style>
+            </Panel.Styles>
+            <TextBlock Text="Funding mode: Automatic" IsVisible="{Binding IsAutomaticSelectionEnabled}" />
+            <TextBlock Text="Funding mode: Manual" IsVisible="{Binding !IsAutomaticSelectionEnabled}" />
```

Comment:
I have no problem with this, but this part of the code was implemented by @soosr and I assume he chose this text for a reason.

Reply:
Funding mode was used in the original design plan by @danwalmsley, that is why I used that.

Reply:
This will probably be removed in future PRs anyhow.

In the meantime, let's stick with this term, we want to avoid "coin" in the default UX.

Reply:
> In the meantime, let's stick with this term, we want to avoid "coin" in the default UX.

Why do we want to avoid the term `coin`?

Even if we have to avoid it we shouldn't replace it by something worse. At least some users understand the term `coin control/selection` but I doubt that anyone understands this `Funding mode`.

Reply:
I agree with @yahiheb, we wanted to avoid the term coin because there were nothing coin related in the software. Since we have coin selection, it is ok to use it.

Reply:
@soosr Also, the privacy ring already mentions the "Wallet coins"

Reply:
I've changed "funding mode" to "coin control" and added a Tooltip. Please, check :)